test/database.yml

```diff
@@ -7,15 +7,13 @@ connections:
       username: <%= ENV["PGUSER"] || "postgres" %>
       password: <%= ENV["PGPASSWORD"] || "" %>
       setup: default
-      schema_search_path: public
     arunit2:
       host: <%= ENV["PGHOST"] || "127.0.0.1" %>
       port: <%= ENV["PGPORT"] || "5432" %>
       database: <%= ENV["PGDATABASE"] || "postgis_adapter_test" %>
       username: <%= ENV["PGUSER"] || "postgres" %>
       password: <%= ENV["PGPASSWORD"] || "" %>
       setup: default
-      schema_search_path: public
```

Comment:
This helps with test like
```
  1) Failure:
AsyncBelongsToAssociationsTest#test_async_load_belongs_to [/Users/safin.rr/.asdf/installs/ruby/3.2.2/lib/ruby/gems/3.2.0/bundler/gems/rails-c694e575cf0f/activerecord/test/cases/associations/belongs_to_associations_test.rb:1871]:
Expected: 1
  Actual: 2
```

But I have no idea why

Reply:
Not sure either. Looking at the original test it does some filtering of the events based on schema name, so maybe adding this search path adds another unexpected query.

https://github.com/rails/rails/blob/dd8f7185faeca6ee968a6e9367f6d8601a83b8db/activerecord/test/cases/associations/belongs_to_associations_test.rb#L1851-L1873

Reply:
But I don't think this is a big deal to remove

Reply:
Contextual note to pair with https://github.com/rgeo/activerecord-postgis-adapter/pull/419#issuecomment-2566620128
This change probably requires some changes in the README.md file since [this section](https://github.com/rgeo/activerecord-postgis-adapter/blob/master/README.md?plain=1#L177) talks about it quite a bit.